src/api/_lib/utils/webhooks.ts

```diff
@@ -37,23 +31,6 @@ export const DiscordWebhooks = useForum ? ForumWebhooks : Webhooks;
 
 export type DiscordWebhooksTarget = keyof typeof DiscordWebhooks | 'none';
 
-export const PerPackageWebhooks: Record<PackageName, DiscordWebhooksTarget> = {
-	actions: 'monorepo',
-	brokers: 'brokers',
-	builders: 'builders',
-	collection: 'collection',
-	'discord.js': 'discord.js',
-	docgen: 'monorepo',
-	guide: 'guide',
-	proxy: 'proxy',
+export const OverideWebhooks: Partial<Record<AppName | PackageName, DiscordWebhooksTarget>> = {
```

Comment:
```suggestion
export const OverrideWebhooks: Partial<Record<AppName | PackageName, DiscordWebhooksTarget>> = {
```

And update the references everywhere else.